common/static/js/src/ReactRenderer.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class ReactRendererException extends Error {
+  constructor (...args) {
+    super(...args);
+    this.name = 'ReactRendererException';
```

Comment:
These errors should log with the `ReactRendererException` name, but for some reason they just display the default generic `Error`. Not blocking, but I'll pick a little more.